src/backend/executor/cypher_set.c

```diff
@@ -113,10 +113,10 @@ static HeapTuple update_entity_tuple(ResultRelInfo *resultRelInfo,
     Buffer buffer;
     bool update_indexes;
     TM_Result   result;
-    CommandId cid = GetCurrentCommandId(true);
-    ResultRelInfo *saved_resultRelInfo = estate->es_result_relation_info;
 
-    estate->es_result_relation_info = resultRelInfo;
+    CommandId cid = GetCurrentCommandId(true);
+    ResultRelInfo **saved_resultRelsInfo  = estate->es_result_relations;
+    estate->es_result_relations = &resultRelInfo;
```

Comment:
Removed a space between variable definitions and the rest of the code. It should be added back in.